src/types.ts

```diff
@@ -3581,6 +3589,12 @@ export interface GameObjRaw {
      * @since v3000.0
      */
     get(tag: Tag | Tag[], opts?: GetOpt): GameObj[];
+    /**
+     * Get a list of all game objs with certain properties.
+     *
+     * @since v3001.0
+     */
+    query(opt: QueryOpt): GameObj[];
```

Comment:
I think query is good. It's short and tells you what it does, it queries the scene for objects. I first thought about just adding a GetOpt as third kind of parameter to get, but it would not go well with the options already passed to get (like recursive).
getQuery sounds like getting a query. The first verb should say what happens, so I think query is best.

Reply:
good